README.md

```diff
@@ -34,10 +34,10 @@ API consumers use the `DroneCore` class to discover and manage vehicles (`Device
 
```

Comment:
I can't mark it up, but docs above mention `Device`: 

>  API consumers use the `DroneCore` class to discover and manage vehicles (`Device` objects),

Reply:
Sorry. Kindly ignore changes in README.md